extensions/cache/src/main/java/org/radargun/stages/cache/background/BackgroundStatisticsManager.java

```diff
@@ -128,7 +128,7 @@ private IterationStats gatherStats() {
             // add zero for all operations we've already reported
             for (Timeline.Category valueCategory : timeline.getValueCategories()) {
                if (valueCategory.getName().endsWith(" Throughput")) {
-                  timeline.addValue(valueCategory, new Timeline.Value(now));
+                  timeline.addValue(valueCategory, new Timeline.Value(0));
```

Comment:
Can you please also change the other 2 calls to new Timeline.Value? Lines 126 and 140. 'now' variable can then be removed altogether.